views/new-steps/add-mentions.js

```diff
@@ -0,0 +1,33 @@
+const { h, computed } = require('mutant')
+const Recipients = require('../component/recipients')
+
+module.exports = function AddMentions ({ state, prev, next, myKey, suggest, avatar }) {
```

Comment:
This is more than just add-mentions, so perhaps it could be named something different.

I need to spin this branch up and have a look at the design to see how this all fits in. It could be that things content could live as part of an existing step ok. Will come back and write

Reply:
this remains as `add-mentions` now